test/integration/discordactions.test.js

```diff
@@ -237,6 +237,28 @@ describe("Discord actions", function () {
       expect(res.body).to.be.an("object");
       expect(res.body.message).to.equal("Role added successfully!");
     });
+    it("should create reson and pass it down to the bot", async function () {
+      fetchStub.returns(
+        Promise.resolve({
+          status: 200,
+          json: () => Promise.resolve({}),
+        })
+      );
+
+      const body = { roleid, userid: userData[0].discordId };
```

Comment:
Isn't this confusing that we are taking discordId of the user and putting it in userId attribute of the body?

Reply:
Yes, it is confusing. will create a ticket to change it. But out of the scope of this ticket.